portal/pom.xml

```diff
@@ -45,7 +45,6 @@
 
   <build>
     <!-- final name of the app -->
-    <finalName>${final-war-name}</finalName>
```

Comment:
There are historical reasons for this, but now, all the maven profiles with exception of heroku can be removed from this pom.